### ISSUE TITLE
About running order

### ISSUE THREAD

**Issue by GuobinZhangTJU** [ICT-MIRACLE-lab/Partially-supervised-multi-organ-segmentation#1]
Hi, many thanks for your sharing, 
Can you provide the running order of your code? the same as nnunet as following?
covert ---  nnUNet_plan_and_preprocess ---- nnUNet_train?
Thanks, waiting for your reply.

**Comment by EricKani** [comment]
Hi,
You can try "python run_multi_training.py 0" directly. # 0 means  GPU number.
Others arguments have been set in script (run_multi_trianing.py).

**Comment by GuobinZhangTJU** [comment]
> Hi,
> You can try "python run_multi_training.py 0" directly. # 0 means GPU number.
> Others arguments have been set in script (run_multi_trianing.py).

Got it, and I am reading your code now, can you help me understand following code:
in nnUNetMultiTrainierV2.py, line 647,
```
        if self.x_tags is None:
            self.x_tags = [tag.lower() for tag in data_dict['tags']]
        y_tags = [tag.lower() for tag in data_dict['tags']]

```
self.x_tags was initialized as 'none', so after executing those lines of code, y_tags will be equal to self.x_tags,
if y_tags is equal to self.x_tags, the loss (show in below) will be the sum of dc_loss and dc_loss  and will not run the 'else' code,
```
    def forward(self, net_output, target, default_task, cur_task):
        if cur_task == default_task:
            # print(f"equal:{cur_task}/{default_task}")
            dc_loss = self.dc(net_output, target)
            ce_loss = self.ce(net_output, target)
            if self.aggregate == "sum":
                result = ce_loss + dc_loss
        else:
            # print(f"not_equal:{cur_task}/{default_task}")
            target_onehot = onehot_transform(target, len(cur_task)+1)
            merged_pre = merge_prediction(
                net_output, target_onehot, cur_task, default_task)
            not_gt = expand_gt(net_output, target_onehot,
                               cur_task, default_task)
            dc_loss = self.dc(merged_pre, target)
            ce_loss = self.ce(merged_pre, target)
            ex_loss = self.ex(net_output, not_gt)

```
Do I understand that right?

I think it should run the 'else' the calculate the 'ex_loss ', but I don'k know how the make  'y_tags' not equal to 'self.x_tags', can you help me ~

thanks again,

**Comment by GuobinZhangTJU** [comment]
I have run your code and my understand is wrong ~  thanks for your share~~~many thanks~~
